src/components/modules/AboutSection/AboutSection.tsx

```diff
@@ -0,0 +1,198 @@
+import { useEffect, useState } from 'react'
+import { useMediaQuery } from 'src/hooks/useMediaQuery'
+
+import { Button } from '@/elements/Button/Button'
+import { NextImage } from '@/elements/NextImage/NextImage'
+
+export const AboutSection = () => {
+  const [active, setActive] = useState(0)
+  const [activeStyle, setActiveStyle] = useState('block')
+  const [btnActive, setBtnActive] = useState(true)
+  const [isRight, setIsRight] = useState(true)
+  const isMobile = useMediaQuery('(max-width: 640px)')
+
+  const STATS = [
+    { name: 'Active Member', count: 700, icon: '/assets/icons/multicultural-people-icon.svg' },
+    { name: 'Material Class', count: 100, icon: '/assets/icons/classroom-icon.svg' },
+    { name: 'Official Partnership', count: 15, icon: '/assets/icons/handshake-icon.svg' },
+  ]
+
+  useEffect(() => {
+    const timer = setInterval(() => (isRight ? handleNextSlide() : handlePrevSlide()), 3000)
+    return () => clearTimeout(timer)
+  })
+
+  const handlePrevSlide = () => {
+    if (btnActive) {
+      setIsRight(false)
+      setBtnActive(false)
+      setActiveStyle('translate-x-[300px]')
+      setTimeout(() => {
+        setActive(active ? --active : STATS.length - 1)
```

Comment:
`--active` sama `++active` bakal nge assign var activenya which is ga boleh. mungkin yg dimaksud active - 1 ?. same as below ya yang `++active`

Reply:
ahhhh yes, oke i'll fix it